locale/en/about/index.md

```diff
@@ -7,7 +7,7 @@ trademark: Trademark
 # About Node.js®
 
 As an asynchronous event-driven JavaScript runtime, Node.js is designed to build
-scalable network applications. In the following "hello world" example, many
+scalable network applications, starting for which doesn't require advanced knowledge of JavaScript, you must be familiar with callback functions and basic JavaScript language. In the following "hello world" example, many
```

Comment:
IMO：You don't need to be so wordy to explain it, but just say something like this following:

As an asynchronous event-driven JavaScript runtime, Node.js is designed to build
scalable network applications. It just needs very basic JavaScript knowledge (See PART 1 at https://javascript.info/).

In the following ……